linux_os/guide/system/accounts/accounts-restrictions/password_storage/accounts_password_all_shadowed_sha512/oval/shared.xml

```diff
@@ -18,7 +18,7 @@
     <filter action="exclude">state_accounts_password_all_shadowed_sha512</filter>
   </unix:shadow_object>
   <unix:shadow_state id="state_accounts_password_all_shadowed_has_no_password" version="1">
-      <unix:password operation="pattern match">^(!|!!|!\*|\*|!locked)$</unix:password>
+      <unix:password operation="pattern match">^(!|!!|!\*|\*|!locked|!\$6\$.*|!!\$6\$.*)$</unix:password>
```

Comment:
I believe it would be better to create another state for "locked password" and add it in the exclusion filter, like in line 17. It would make it more transparent since "no password" is different than "locked password".

Could you update this, please?

Reply:
added a exclusion filter, but not really sure about the name/id